### PR TITLE
KOGITO-4255 - Fix GrafanaDockerComposeIT in dmn-drools-springboot-metrics example

### PR DESCRIPTION
**Pull request by r00ta**
JIRA: https://issues.redhat.com/browse/KOGITO-4255

All the services have to be ready before the tests are run. 

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java [hunk 1]
```diff
@@ -45,17 +47,24 @@
     static {
         try {
             environment = new DockerComposeContainer(new File(GrafanaDockerComposeIT.class.getClassLoader().getResource("./docker-compose.yml").toURI()))
-                    .withExposedService("grafana_1", 3000, Wait.forListeningPort().withStartupTimeout(Duration.ofMinutes(8)));
+                    .withExposedService("grafana_1", 3000, Wait.forListeningPort().withStartupTimeout(Duration.ofMinutes(STARTUP_MINUTES_TIMEOUT)))
```
Comment (jiripetrlik) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java]: Maybe a detail. Can we make the numbers of ports constants and move it upward. It seems that these numbers are used on multiple places.

**Review by jiripetrlik — APPROVED** [review]
One small comment. I was able to run the test locally.

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by danielezonca** [comment]
Let's run a couple of times to check the stability

**Pull request merged**